INSTALL

```diff
@@ -27,8 +27,8 @@ The source for the file utility + library is available from
     ftp://ftp.astron.com/pub/file/
 
 You will need a cryptographic library to support digests and
```

Comment:
It gets tricky here because now those two are separated in some of the options. What a crazy tangle :smile: But yeah the language here needs some more love. I also realized the "Use of rpm-sequoia..." paragraph looks odd where it is now  - I just moved it a bit earlier without other changes but that doesn't quite cut it.